### ISSUE TITLE
Plugin not found

### ISSUE THREAD

**Issue by looo87** [markstory/asset_compress#303]
So I followed the documentation but cant get it to work and it might be because of my folders management. 

I have something like this 

Project
  |-- Cakephp 1
     |-- app
        |-- Plugin (here I have AssetCompress)
  |-- Cakephp 2
     |-- app
        |-- Plugin (here I have AssetCompress)
  |-- Cakephp 3
     |-- app
        |-- Plugin (here I have AssetCompress)
  |-- Library ( here is my ShellDispatcher)

My config looks right:
       Project/Cakephp(1,2,3)/app/Config/asset_compress.ini
               [General]
               cacheConfig = true
               timestamp = true

              [css]
              path = WEBROOT/app/webroot/css/*
              cacheFiles = false
              timestamp = true
              cachePath = WEBROOT/cache_css/
              filters[] = YuiCss

              [this.css]
              files[] = this.css

        project/cakephp(1,2,3)/app/Config/bootstrap.php
              CakePlugin::load('AssetCompress', array('bootstrap' => true));

        Project/Cakephp(1,2,3)/app/Controller/AppController.php
                public $helpers = ['AssetCompress.AssetCompress'];
               
       view.ctp 
                <?php echo $this->AssetCompress->css('this') ?>
             --> outputs:
                  <link rel="stylesheet" type="text/css" ref="https://www.example.com/cache_css/this.css">

So am I doing something wrong or do I have to ajust something?
Thank you in advance!

**Comment by markstory** [comment]
I'm not sure what's going on. Do you have 3 different cakephp apps in a single project all using different versions of CakePHP?

**Comment by looo87** [comment]
No, I have the same version of cakephp in all 3 cases, but the library (or lib -- with shelldispatcher) is not present in every instance of cake. It's in the root folder of the project and I think it was made like this to not have shelldispatcher multiple times.

**Comment by markstory** [comment]
Not sure then. I've not come across an application structure like that before.

**Comment by agarzon** [comment]
Having similar problem.

Cake 3.5.12

```
$ composer require markstory/asset_compress
```

Then in bootstrap.php
```
Plugin::load('AssetCompress', ['bootstrap' => true]);
```

I get:
```
Missing Plugin Cake\Core\Exception\MissingPluginException
Error: The application is trying to load a file from the AssetCompress plugin.
Make sure your plugin AssetCompress is in the /app/plugins/ directory and was loaded. 
```

And from CLI:

```
Exception: Plugin AssetCompress could not be found. in [/home/agarzon/projects/ppc/vendor/cakephp/cakephp/src/Core/Plugin.php, line 149]
```
